redshift/aws_lambda/lambda_function.py

```diff
@@ -50,10 +50,10 @@ def lambda_handler(request, context):
     # Make sure you should know these details about the table you are migrating beforehand
     # Set the "limit" according to your estimates of the table's size and row count
     # Again, these can also be stored in "request"
-    table = request["table"]
-    primary_key = request["primary_key"]
-    cursor = request["cursor"]
-    limit = request["limit"]
+    table = request['secrets']["table"]
+    primary_key = request['secrets']["primary_key"]
```

Comment:
These double-quotes should also be single-quotes.

Reply:
oops